sdks/fabric/interoperation-node-sdk/makefile

```diff
@@ -15,11 +15,13 @@ build-local:
 		cp -r ../../../common/protos-js/peer ./protos-js/ && \
 		cp -r ../../../common/protos-js/relay ./protos-js/ && \
 		cp -r ../../../common/protos-js/package.json ./protos-js/
-	cp package.json package-remote.json && \
+	(cp package.json package-remote.json && \
 		cp package-local.json package.json && \
 		npm install && \
-		mv package-remote.json package.json && \
-	npm run build && cp -r protos-js build/
+		npm run build && \
+		cp -r protos-js build/) || true
```

Comment:
Again, is it OK if this fails?

Reply:
Its not okay to fail, its just if it fails or succeeds, the `package.json` is reverted back in any case.
Otherwise we used to see `package.json` was remained to be a copy of `package-local.json`.